guides/hhvm/01-installation/06-release-schedule.md

```diff
@@ -1,4 +1,4 @@
-Since HHVM 4.2, we aim to release a new release weekly, promoting a recent
+We release a new release biweekly (every two weeks), promoting a recent
```

Comment:
How about "We release biweekly"? Releasing a release seems slightly redundant :)